packages/mirinae/src/foundation/icons/p-icons/ic_filter.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
+/* tslint:disable */
+// @ts-ignore
+import icon from 'vue-svgicon'
+icon.register({
+  'ic_filter': {
+    width: 16,
+    height: 16,
+    viewBox: '0 0 16 16',
+    data: '<path pid="0" fill-rule="evenodd" clip-rule="evenodd" d="M5 1a2 2 0 100 4 2 2 0 000-4zm0 2.7a.7.7 0 100-1.4.7.7 0 000 1.4z" _fill="#232533"/><path pid="1" fill-rule="evenodd" clip-rule="evenodd" d="M14.5 3.75a.5.5 0 00.5-.5v-.5a.5.5 0 00-.5-.5h-13a.5.5 0 00-.5.5v.5a.5.5 0 00.5.5h13zM5 3.7a.7.7 0 100-1.4.7.7 0 000 1.4zM5 11a2 2 0 100 4 2 2 0 000-4zm0 2.7a.7.7 0 100-1.4.7.7 0 000 1.4z" _fill="#232533"/><path pid="2" fill-rule="evenodd" clip-rule="evenodd" d="M14.5 13.75a.5.5 0 00.5-.5v-.5a.5.5 0 00-.5-.5h-13a.5.5 0 00-.5.5v.5a.5.5 0 00.5.5h13zM5 13.7a.7.7 0 100-1.4.7.7 0 000 1.4zM11 10a2 2 0 100-4 2 2 0 000 4zm0-2.7a.7.7 0 100 1.4.7.7 0 000-1.4z" _fill="#232533"/><path pid="3" fill-rule="evenodd" clip-rule="evenodd" d="M1.5 7.25a.5.5 0 00-.5.5v.5a.5.5 0 00.5.5h13a.5.5 0 00.5-.5v-.5a.5.5 0 00-.5-.5h-13zm9.5.05a.7.7 0 100 1.4.7.7 0 000-1.4z" _fill="#232533"/>'
+  }
+})
```

Comment:
Actually, this icon usage is not vue3's one. But, leave it as it is, I'll fix it later..